protocol/messenger_response.go

```diff
@@ -489,6 +502,20 @@ func (r *MessengerResponse) AddRemovedChat(chatID string) {
 	r.removedChats[chatID] = true
 }
 
+func (r *MessengerResponse) AddRemovedActivityCenterNotifications(notifications []string) {
+	for _, n := range notifications {
+		r.AddRemoveActivityCenterNotification(n)
+	}
+}
+
+func (r *MessengerResponse) AddRemoveActivityCenterNotification(notificationID string) {
```

Comment:
Tiny typo, should probably be `AddRemovedActivityCenterNotification` (missing `d`)